src/main/java/com/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,22 @@
+package com.cinema.controller;
+
+import com.cinema.model.dto.UserAuthenticationDto;
+import com.cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserAuthenticationDto userAuthenticationDto) {
```

Comment:
UserAuthenticationDto and UserRegistrationDto could be different. For example, in the `UserRegistrationDto` you could have fields  `login`, `password`, and `repeatPassword`. But in `UserAuthenticationDto` you could have only `login` and `password`.